bait-files/src/test/java/com/google/sps/LoginTest.java

```diff
@@ -0,0 +1,59 @@
+package com.google.sps;
+
+import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+import static com.google.sps.data.Keys.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.*;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Test;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalUserServiceTestConfig;
+import java.io.*;
+import javax.servlet.http.*;
+import javax.servlet.ServletException;
+import com.google.sps.servlets.UsernameTakenServlet;
+import com.google.sps.servlets.LoginServlet;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class LoginTest {
```

Comment:
Nit: Include the word "Servlet" in this class and file name: LoginServletTest.  To make it super obvious that the class you are testing is LoginServlet.

Reply:
Rename to servlet